include/private/meta/sampler.h

```diff
@@ -33,6 +33,11 @@ namespace lsp
         // Sampler metadata
         struct sampler_metadata
         {
+            static const float SAMPLE_RATE_MIN        = 44.1f;     // Minimum rate (ms)
```

Comment:
This is not beautiful. When adding constants, try to align all parts of the expression as it is done below.